docs/guide/personalization.md

```diff
@@ -265,294 +264,285 @@ For the `content` interface one Content item is returned, for the `contenttypes`
 
-After defining which Content Types should be tracked and recommended  you can start the full export by executing the following command:
+After [defining which Content Types should be tracked and recommended](#set-up-content-type-tracking),
+start the full export by executing the following command:
 
 ``` bash
-php app/console ezreco:runexport --contentTypeIdList=<contentTypeId>,<contentTypeId> --webHook=https://admin.yoochoose.net/api/<your_customer_id>/items --hidden=1 --mandatorId=<your_customer_id> --host=<your_ezplatform_host_with_scheme>
+php bin/console ezrecommendation:export:run
+    --contentTypeIdList=<contentTypeId>,<contentTypeId>
+    --webHook=https://admin.yoochoose.net/api/<your_customer_id>/items
+    --hidden=1 --mandatorId=<your_customer_id>
+    --host=<your_ezplatform_host_with_scheme>
 ```
 
-With this command, the bundle exporter will collect all content related to the SiteAccesses of this customerID and store it in files (1).
-After finishing, the systems will send a POST request to the `webHook` endpoint and inform the personalization engine to fetch new content (2).
-An internal workflow is then triggered (3) so that the generated files are downloaded (4) and imported in the personalization engine's content store (5).
+With this command, the bundle exporter collects all content related to the SiteAccesses of this customerID and stores it in files (1).
+After finishing, the systems sends a POST request to the `webHook` endpoint and informs the Recommendation engine to fetch new content (2).
+An internal workflow is then triggered (3) so that the generated files are downloaded (4) and imported in the Recommendation engine's content store (5).
 
-Please be patient, as this can take up to a couple of minutes.
+The export process can take up to a few minutes.
 
 ![Recommendation Full Content Export](img/full_content_export.png)
 
-!!! caution "Changing Content Types for recommendations"
+!!! caution "Re-exporting modified Content Types"
 
-    If the Content Types to be recommended are changed, a full export needs to be started again by running the `php app/console ezreco:runexport` command.
+    If the Content Types to be recommended are changed, you need to perform a new full export
+    by running the `php bin/console ezrecommendation:export:run` command again.
 
-### Checking if the full context export was stored in the recommender engine
+### Checking export results
 
-There are three ways to check if content was transferred and stored successfully in the recommender engine:
+There are three ways to check if content was transferred and stored successfully in the Recommendation engine:
 
 #### REST request to the recommender's content store
 
-To get the content of an imported item you can request the following REST resource
+To get the content of an imported item you can request the following REST resource:
 
 `GET https://admin.yoochoose.net/api/<your_customer_id>/item/<your_content_type>/<your_content_id>`
```

Comment:
Add "_id" at the end of "your_content_type" parameter (should be: your_content_type_id)